### PR TITLE
*: Cut 0.22.0

### PR DESCRIPTION
**Pull request by mxinden**

### CODE REVIEW

**Comment by mxinden** [comment]
@brancz do you see any version bumps missing by the build system?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 * [CHANGE] Deprecate specification of Prometheus rules via ConfigMaps in favor of `PrometheusRule` CRDs
 
+## 0.22.0 / 2018-07-09
+
+* [FEATURE] Allow seting volume name via volumetemplateclaimtemplate in prom and alertmanager (#1538)
```
Comment (squat) [inline CHANGELOG.md]: s/seting/setting

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+* [FEATURE] Allow settings custom tags of container images (#1584)
```
Comment (squat) [inline CHANGELOG.md]: s/settings/setting

**Comment by mxinden** [comment]
@squat thanks for catching those!

**Comment by brancz** [comment]
lgtm :+1:

**Pull request merged**